services/packagist/packagist-php-version.spec.js

```diff
@@ -7,20 +7,20 @@ const PackagistPhpVersion = require('./packagist-php-version.service')
 describe('PackagistPhpVersion', function () {
   const json = {
     packages: {
-      'frodo/the-one-package': {
-        '1.0.0': { require: { php: '^5.6 || ^7' } },
-        '2.0.0': { require: { php: '^7.2' } },
-        '3.0.0': { require: { php: '^7.4 || 8' } },
-        'dev-main': { require: { php: '^8' }, 'default-branch': true },
-      },
-      'samwise/gardening': {
-        '1.0.x-dev': {},
-        '2.0.x-dev': {},
-      },
-      'pippin/mischief': {
-        '1.0.0': {},
-        'dev-main': { require: {}, 'default-branch': true },
-      },
```

Comment:
it typically concerns me when we have to modify tests as part of some other change. I realize that _some_ updates are inevitable here since the new api has a different response structure, but I feel like there's more going on or at least that I'm missing some context (though I presume this is all branch related)

Reply:
This change is solely because of the change in response structure as seen in the diff.